source/MultiGrid/MultiGrid.js

```diff
@@ -139,9 +139,14 @@ export default class MultiGrid extends PureComponent {
         rowIndex,
       });
 
+    let oldLeftGridWidth = this._leftGridWidth;
     this._leftGridWidth = null;
     this._topGridHeight = null;
     this._maybeCalculateCachedStyles(null, this.props, null, this.state);
+    if (oldLeftGridWidth != this._leftGridWidth)
```

Comment:
The same problem exists when there is fixed rows and their height changes. If this code checked both the `leftGridWidth` and the `topGridHeight` it would handle both correctly. 

```diff
diff --git a/source/MultiGrid/MultiGrid.js b/source/MultiGrid/MultiGrid.js
index 74675e6..c98fe15 100644
--- a/source/MultiGrid/MultiGrid.js
+++ b/source/MultiGrid/MultiGrid.js
@@ -139,12 +139,12 @@ export default class MultiGrid extends PureComponent {
         rowIndex,
       });
 
-    let oldLeftGridWidth = this._leftGridWidth;
+    const oldLeftGridWidth = this._leftGridWidth;
+    const oldTopGridHeight = this._topGridHeight;
     this._leftGridWidth = null;
     this._topGridHeight = null;
     this._maybeCalculateCachedStyles(null, this.props, null, this.state);
-    if (oldLeftGridWidth != this._leftGridWidth)
-    {
+    if (oldLeftGridWidth != this._leftGridWidth || oldTopGridHeight != this._topGridHeight) {
       this.forceUpdate();
     }
   }
```

Here is a test case that would cover the fixed row height changing: https://gist.github.com/petersendidit/a80d38510a2d752a231715061ab3eab1